.github/workflows/publish.yml

```diff
@@ -72,3 +72,29 @@ jobs:
           push: true
           tags: ${{ steps.docker_metadata.outputs.tags }}
           labels: ${{ steps.docker_metadata.outputs.labels }}
+
+  dockerhub-description:
+    name: upload readme as dockerhub image description
```

Comment:
Very minor suggestion (for consistency):

```suggestion
    name: Upload readme as dockerhub image description
```

Reply:
Sorry. After checking, the rule is to **not** specify a `name` for the `job`. Thus the jobs are named with their kebab key.

<img width="240" alt="image" src="https://user-images.githubusercontent.com/9574336/185360710-67124730-d04a-4aeb-8966-ccc24d3b2d00.png">